src/analytics/AnalyticsPage.react.js

```diff
@@ -5,20 +5,56 @@
 
 import 'AnalyticsPage.react.css';
 
+import {Typography} from '@mui/material';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import Container from '@mui/material/Container';
+import CssBaseline from '@mui/material/CssBaseline';
```

Comment:
```suggestion
import ScopedCssBaseline from '@mui/material/ScopedCssBaseline';
```

As we discussed before, scoped (local) CSS base line is preferred over the global one.